snippets/header-drawer.liquid

```diff
@@ -54,8 +58,11 @@
                                     {{ childlink.title | escape }}
                                   </a>
                                 {%- else -%}
-                                  <details id="Details-menu-drawer-submenu-{{ forloop.index }}">
```

Comment:
I noticed this existing ID was also resulting in duplicate ids, so I've updated the id to use a similar format as the other link elements ids I'm adding this PR. This was also an issue for the grandchild child below, and in the dropdown menu.